Makefile

```diff
@@ -43,4 +43,4 @@ build-trace-docker:
 
 test:
 	mkdir -p coverage
-	@go test -race -short -count=1 -coverprofile=coverage/coverage.cov ${GO_PKG_LIST}
```

Comment:
Why was this removed?

Reply:
because according to the docs, -race is not even a valid flag for go test. It is only a go build flag. Unless you know differently...

Reply:
well, now I found another doc that says it is available for build, clean, get, install, list, run, and test commands. I'll put it back.

Reply:
but I guess that begs the question as to why it is only on make test, and not make build also.

Reply:
Build flags are valid for running tests. I think we should keep it. I have found value in the -race flag when writing tests with channels and go routines.